### PR TITLE
class02-hw3-YuNyuk

### PR DESCRIPTION
**Pull request by YuNyuk**

### CODE REVIEW

File: class02/homework/YuNyuk/hw3/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+./splitted_dataset/train/<good>: 6​
+```
+
+## Training 결과
+|Classification model|Accuracy|FPS|Training time|Batch size|Learning rate|Other hyper-prams|
+|----|----|----|----|----|----|----|
+|EfficientNet-V2-S|----|----|----|----|----|----|
+|EfficientNet-B0| ----|----|----|----|----|----|
+|DeiT-Tiny| ----|----|----|----|----|----|
+|MobileNet-V3-large-1x|1.000|----|0:00:06.600|48|0.0058|----|
```
Comment (JerryJLee) [inline class02/homework/YuNyuk/hw3/README.md:22]: 모델 하나만 성공?

**Review by JerryJLee — APPROVED** [review]
